pkg/cmd/server/origin/admission/sync_test.go

```diff
@@ -31,7 +31,6 @@ var admissionPluginsNotUsedByKube = sets.NewString(
 	"RunOnceDuration",                        // from origin, used for overriding the ActiveDeadlineSeconds for run-once pods
 
 	"NamespaceExists",  // superseded by NamespaceLifecycle
-	"InitialResources", // do we want this? https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/initial-resources.md
```

Comment:
I don't think we want that. @derekwaynecarr, you aware of this?

Reply:
We don't want this.  It never made it past alpha, and the new VPA mechanism replaces it.